packages/docusaurus-plugin-content-docs/index.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+const globby = require('globby');
+const {getSubFolder, idx, normalizeUrl} = require('@docusaurus/utils');
+const createOrder = require('./src/order');
+const loadSidebars = require('./src/sidebars');
+const processMetadata = require('./src/metadata');
+
+const DEFAULT_OPTIONS = {
+  metadataKey: 'docsMetadata',
+  metadataFileName: 'docsMetadata.json',
+  path: 'docs', // Path to data on filesystem, relative to site dir.
+  routeBasePath: 'docs', // URL Route.
+  include: ['*.md', '*.mdx'], // Extensions to include.
+  // TODO: Read from props rather than hardcoded sidebar.json.
+  sidebar: [], // Sidebar configuration for showing a list of documentation pages.
+  docLayoutComponent: '@theme/Doc',
+  docItemComponent: '@theme/DocBody',
```

Comment:
Upon some thinking. I realize that the theme component is inside docusaurus own package. What if user doesn't use plugin-docs. It's like an extra component exported here.

Maybe write it as a TODO for now. We haven't really settled how we export theme and UI related components

Reply:
This is just a string. It's not doing any import/export whatsoever. Users not using this plugin will not import the component. This doesn't have to be a TODO because we don't have to change anything here (I hope).

Reply:
Hmm it is true that user not using this plugin will not import the component. But I realize that we have this `DocBody` and `Doc` inside `docusaurus` package. What if they don't use the docs plugin ? The component is still installed as part of 'docusaurus' package.

I'm still not clear on how we're going to do the theming/ providing UI packages to user